src/Workspaces/Core/Portable/Options/ExportOptionAttribute.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.ComponentModel.Composition;
+using System.Composition;
```

Comment:
Why? The only places we had the option service actually having to discover all options was mostly in test code. I wouldn't see why this would be any different.